urdf_parser/include/urdf_parser/urdf_parser.h

```diff
@@ -48,18 +48,19 @@
 
 #include "exportdecl.h"
 
+using TiXmlDocument = tinyxml2::XMLDocument;
+using TiXmlElement = tinyxml2::XMLElement;
```

Comment:
I'd really rather not pretend that tinyxml2 APIs are the same as the tinyxml ones; this is changing the API below, but in a sneaky way.  Let's remove this, deprecate the tinyxml APIs, and also add in the tinyxml2 APIs.